.circleci/config.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+jobs:
+  build:
+    machine: true
+    steps:
+    - checkout
+
+    - run: |
+        wget -q $(curl -sS -H "Authorization: token $RELEASE_TOKEN" https://api.github.com/repos/giantswarm/architect/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
+        chmod +x ./architect
+        ./architect version
+    - run: ./architect build
+    - deploy:
```

Comment:
If we are planning to run test jobs in this repo it would be good to define a workflow to begin with and have a separate job for deploying, like https://github.com/giantswarm/chart-operator/blob/master/.circleci/config.yml#L50-L62

Reply:
Ack, this here is only to make it build so far. I can imagine there will be e2e tests for the lib itself but I am not sure how this will look like. I think there will not be any deployment but I will revisit this later on. Don't want to bother with this now. But yes, we will have workflows then. Thanks for the feedback.